src/util.ts

```diff
@@ -35,3 +35,14 @@ export const shortString = (st : string, n: number) => st.length > n ? st.substr
 export function isCarRequest(request : Request) {
   return request.request.headers.some((header) => header.name.toLowerCase() == 'content-type' && header.value == CAR.contentType)
 }
+
+export function formatError(error: any): string {
+  if (error === null || error === undefined) {
+    return "Error: null or undefined";
+  }
+  try {
+    return JSON.stringify(error, null, 2); // Format JSON with indentation
```

Comment:
**suggestion:** Looks like we're using 2-space indentation here, but 4-space indentation on `Out:`. Should we standardize? 2 might be the better choice here, since space (hah!) is limited. Plus it's more common in JS anyhow.

Reply:
agree :)

Reply:
Good catch. I've updated it to 2.